### ISSUE TITLE
Fix Nx Cloud Report warnings (We didn't find any information for the current pull request ...)

### ISSUE THREAD

**Issue by jonrkarr** [reproducible-biomedical-modeling/CRBM-Viz#2733]
I think this can be fixed by replacing the `NX_BRANCH` environment variable for each job 
```yaml
...
jobs:
  getAffectedLibrariesApps:
    env:
        NX_BRANCH: ${{ github.event.pull_request.head.ref }}  
...
```

with a global variable
```yaml
env:
  NX_BRANCH: ${{ github.event.pull_request.head.ref }}  
```

I wasn't sure if @bilalshaikh42 intentionally set up this environmental variable this way.

**Comment by bilalshaikh42** [comment]
This warning is expected the first time a branch runs as there will be no information on the cloud builder. After, the first run, you should see a report such as this:  https://github.com/biosimulations/Biosimulations/pull/2725#issuecomment-883949120. Are you getting this warning elsewhere?

**Comment by jonrkarr** [comment]
That's when I seeing the warning. Wasn't sure if this was expected.
